zhmcclient/_lpar.py

```diff
@@ -500,6 +501,11 @@ def load(self, load_address=None, load_parameter=None,
             TBD: What will happen with the LPAR in that case (deactivated then
             activated? nothing?)
 
+          clear_indicator (bool):
+            Optional boolean controlling whether the memory should be
+            cleared before performing the load or not cleared. The
+            default value is `True`.
+
```

Comment:
I suggest to move the "clear_indicator" argument to before the "wait_for_completion" argument. It is one that is put into the request body, and those we usually have before those that the zhmcclient adds.

Reply:
Okay. I'll move "store_status_indicator" parameter as well.